src/content/learn/react-compiler.md

```diff
@@ -126,29 +126,46 @@ In addition to these docs, we recommend checking the [React Compiler Working Gro
 
 ### Installing eslint-plugin-react-compiler {/*installing-eslint-plugin-react-compiler*/}
 
-React Compiler also powers an eslint plugin. The eslint plugin can be used **independently** of the compiler, meaning you can use the eslint plugin even if you don't use the compiler.
+React Compiler also powers an ESLint plugin. The ESLint plugin can be used **independently** of the compiler, meaning you can use the ESLint plugin even if you don't use the compiler.
 
 <TerminalBlock>
 npm install -D eslint-plugin-react-compiler@beta
 </TerminalBlock>
 
-Then, add it to your eslint config:
+Then, add it to your ESLint config:
+
+```js
+import reactCompiler from 'eslint-plugin-react-compiler'
+
+export default [
+  {
+    plugins: {
+      'eslint-plugin-react-compiler': reactCompiler,
+    },
+    rules: {
+      'react-compiler/react-compiler': 'error',
```

Comment:
I think this should be:

```js
'eslint-plugin-react-compiler/react-compiler': 'error'
```

If not, you will get the following error:

```sh
[Error] Request textDocument/diagnostic failed.
  Message: Request textDocument/diagnostic failed with message: Key "rules": Key "react-compiler/react-compiler": Could not find plugin "react-compiler".
  Code: -32603 
[Error] Document pull failed for text document file:///.../eslint.config.mjs
  Message: Request textDocument/diagnostic failed with message: Key "rules": Key "react-compiler/react-compiler": Could not find plugin "react-compiler".
  Code: -32603
  ```

Reply:
No, that's not how it works - check other plugin docs:

- https://www.npmjs.com/package/eslint-plugin-react-x

Reply:
Yea @AnthonyLzq is right here

in flat config you can infer the plugin namespace in the plugins object [ref here](https://eslint.org/docs/latest/use/configure/plugins#configure-plugins)

in the example you shared the plugin namespace is `react-x` thats why rules are referred with `react-x/`

Reply:
sorry @AnthonyLzq @harish-sethuraman I see what you mean now

I fixed it the more conventional way, by using `'react-compiler'` as a key in the `plugins` object:

- 426a8abb04a943295e4a5e403a2bc14d42ceb04c